_includes/configuration/content-langs.md

```diff
@@ -0,0 +1,69 @@
+## `content_langs`
```

Comment:
```suggestion
## content_langs
```
Not sure on this one, as some files have it in code but others don't. I believe this shouldn't be marked up as code though.

Reply:
Might see what Tyler says on this for now.